interfaces/cython/cantera/ck2yaml.py

```diff
@@ -1328,6 +1336,33 @@ def parse_expression(expression, dest):
 
         return reaction, revReaction
 
+    def load_extra_file(self, path):
+        """
+        Load YAML-formatted entries from ``path`` on disk.
+        """
+        with open(path, 'rt', encoding="utf-8") as stream:
+            yml = yaml.round_trip_load(stream)
+
+        # do not overwrite reserved field names
+        reserved = {'generator', 'input-files', 'cantera-version', 'date',
+                    'units', 'phases', 'species', 'reactions'}
+        reserved &= set(yml.keys())
+        if reserved:
+            raise InputError("A YAML file provided as '--extra' input "
+                "(i.e. '{}') must not redefine reserved field names: "
+                "{}".format(path, reserved))
```

Comment:
I would like to clarify this a little bit, but I can't give a multi-line suggestion. What about:
```
            raise InputError("The YAML file '{}' provided as '--extra' input "
                "must not include the reserved field name: "
                "'{}'".format(path, reserved))
```